ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -7390,6 +7393,76 @@ router ospf 600
    area 0.0.20.26 nssa default-information-originate metric 50 metric-type 1 nssa-only
 ```
 
+### IPv6 Router OSPF
+
+#### IPv6 Router OSPF Summary
```

Comment:
we need VRF in this table, and probably also the redistribution table. Otherwise it becomes really hard to track.

Reply:
Added!